extensions/extensions-helm/aissemble-quarkus-chart/values.template.yaml

```diff
@@ -20,7 +20,7 @@ deployment:
   volumes:
     - name: quarkus-application-properties
       configMap:
-        name: supplemental-quarkus-config
+        name: "{{ .Values.configMap.name }}"
```

Comment:
I: Set's the config map name to the value of `.Values.configMap.name` which can then be easily overwritten in downstream helm charts.

Reply:
So the `tpl` call in the template processes this bit as a template and so expands this to the actual value instead of using the string literally?

Reply:
Correct.